powerauth-push-server/src/main/java/io/getlime/push/controller/rest/CampaignController.java

```diff
@@ -229,9 +230,9 @@ public Response addUsersToCampaign(@PathVariable(value = "id") Long campaignId,
      * @param request List of users to remove
      * @return Response status
      */
-    @RequestMapping(value = "{id}/user/remove")
+    @RequestMapping(value = "{id}/user/remove", method = RequestMethod.PUT)
```

Comment:
- Use `{id}/user/delete` instead of `{id}/user/remove` to keep the naming consistent.
- Why `PUT` method? Let's use `POST` for deleting...